lib/KTable/index.vue

```diff
@@ -0,0 +1,480 @@
+                  return;
+                } else {
+                  if (colIndex < totalCols - 1) {
+                    nextColIndex = colIndex + 1;
+                  } else if (rowIndex < totalRows - 1) {
+                    nextColIndex = 0;
+                    nextRowIndex = rowIndex + 1;
+                  } else {
+                    nextColIndex = 0;
+                    nextRowIndex = -1;
+                  }
+                }
+              } else {
+                if (focusedElementIndex < focusableElements.length - 1) {
+                  focusableElements[focusedElementIndex + 1].focus();
+                  event.preventDefault();
+                  return;
+                } else {
+                  if (colIndex > 0) {
+                    nextColIndex = colIndex - 1;
+                  } else if (rowIndex > 0) {
+                    nextColIndex = totalCols - 1;
+                    nextRowIndex = rowIndex - 1;
+                  } else {
+                    nextColIndex = totalCols - 1;
+                    nextRowIndex = totalRows - 1;
+                  }
+                }
+              }
+            } else {
+              if (!event.shiftKey) {
+                if (colIndex < totalCols - 1) {
+                  nextColIndex = colIndex + 1;
+                } else if (rowIndex < totalRows - 1) {
+                  nextColIndex = 0;
+                  nextRowIndex = rowIndex + 1;
+                } else {
+                  nextColIndex = 0;
+                  nextRowIndex = -1;
+                }
+              } else {
+                if (colIndex > 0) {
+                  nextColIndex = colIndex - 1;
+                } else if (rowIndex > 0) {
+                  nextColIndex = totalCols - 1;
+                  nextRowIndex = rowIndex - 1;
+                } else {
+                  nextColIndex = totalCols - 1;
+                  nextRowIndex = totalRows - 1;
+                }
+              }
+            }
+            this.focusCell(nextRowIndex, nextColIndex);
+            this.focusedRowIndex = nextRowIndex === -1 ? null : nextRowIndex;
+            this.focusedColIndex = nextColIndex;
+            this.highlightHeader(nextColIndex);
+            event.preventDefault(); // Prevent default Tab behavior
+            break;
+
+          default:
+            return;
+        }
+
+        this.focusCell(nextRowIndex, nextColIndex);
+        this.focusedRowIndex = nextRowIndex === -1 ? null : nextRowIndex;
+        this.focusedColIndex = nextColIndex;
+
+        this.highlightHeader(nextColIndex);
+
+        event.preventDefault();
+      },
+      getCell(rowIndex, colIndex) {
+        if (rowIndex === -1) {
+          return this.$el.querySelector(`thead th:nth-child(${colIndex + 1})`);
+        } else {
+          return this.$el.querySelector(
+            `tbody tr:nth-child(${rowIndex + 1}) td:nth-child(${colIndex + 1})`
+          );
+        }
+      },
+      scrollCellIntoView(cell) {
```

Comment:
Sure, I will!

Reply:
There is a performance issue here, but not because of layout or repainting, but because of scripting. Having more than 2 seconds scripting in a test of 6 seconds just tabbing at a normal speed like an user could do is a lot.

![image](https://github.com/user-attachments/assets/c9bc1caf-8387-40c9-9e61-ec67afa53669)

This is proably because we have several querySelectors to get the current cell, to get the focusable elements inside the cell, to get the next cell, to get the sticky header, to get the sticky column, and to get the header to highlight. Besides with the fact that querySelector and querySelectorAll has a less performance in several cases compared to the traditionals getElementBy* https://dev.to/wlytle/performance-tradeoffs-of-queryselector-and-queryselectorall-1074. 

I think replacing all of these calls with [vue template refs](https://www.w3schools.com/vue/vue_refs.php) will help a lot with this

Reply:
Thanks for looking into this @AlexVelezLl, this is a great catch.

Reply:
Hey @AlexVelezLl,
I made the changes in replacing queryselectors with getElementbyTagName and Vue refs. Can you please check now if the performance is better now?

Reply:
Hey! Just tested it, and this is working so much better! Thank you @BabyElias!!